### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -283,7 +283,7 @@ if test x"$request_openssl" = "xcheck"; then
 else
     OPENSSL_BIN="$request_openssl/bin/openssl"
     LDFLAGS="$LDFLAGS -L$request_openssl/lib";
-    CFLAGS="$CFLAGS -I$request_opsnssl/include";
+    CFLAGS="$CFLAGS -I$request_openssl/include";
```
Comment (jsoref) [inline configure.ac]: It's possible this is from an upstream.

File: test/test_0310_conf_store.py [hunk 2]
```diff
@@ -275,16 +275,16 @@ def test_310_202(self):
 
     def test_310_203(self):
         # test case: remove one md, keep another
-        dnsList1 = ["greenybtes2.de", "www.greenybtes2.de", "mail.greenybtes2.de"]
+        dnsList1 = ["greenbytes2.de", "www.greenbytes2.de", "mail.greenbytes2.de"]
```
Comment (jsoref) [inline test/test_0310_conf_store.py]: As a general rule, when I encounter pairs of similar tokens, I assume that the lesser used/lesser reasonable one is a typo. If this is an intentional thing, I'd suggest using something that clearly indicates that intention.

**Comment by icing** [comment]
Thanks! Appreciate it. I am terribly bad at catching those...

**Comment by jsoref** [comment]
Please give my tool(s) a try -- I'm happy to improve them based on feedback. They should make dealing w/ spelling much easier. Especially (in theory) `fchurn` since it will only show you newly introduced tokens.

**Pull request merged**